pmd-java/src/main/java/net/sourceforge/pmd/lang/java/types/TypeTestUtil.java

```diff
@@ -134,26 +134,35 @@ public static boolean isA(@NonNull String canonicalName, @Nullable JTypeMirror t
         return isA(canonicalName, thisType, null);
     }
 
+    public static boolean isA(@NonNull JTypeMirror t1, @Nullable TypeNode t2) {
+        return t2 != null && isA(t1, t2.getTypeMirror());
+    }
+
     /**
-     * This is the subtyping routine we use, which prunes some behavior
-     * of isSubtypeOf that we don't want (eg, that unresolved types are
-     * subtypes of everything).
+     * Checks whether the first type is a subtype of the second. This
+     * removes some behavior of isSubtypeOf that we don't want (eg, that
+     * unresolved types are subtypes of everything).
+     *
+     * @param t1 A supertype
+     * @param t2 A type
+     *
+     * @return Whether t1 is a subtype of t2
      */
-    private static boolean isA(JTypeMirror t1, JTypeMirror t2) {
-        if (t1 == null || t2 == null) {
+    private static boolean isA(@Nullable JTypeMirror t1, @NonNull JTypeMirror t2) {
```

Comment:
I reversed the order of the arguments here because all other overloads of `isA` expect the supertype as the first parameter and the subtype as the second.